### PR TITLE
Add Heptapod to Source Code Repos

### PR DESCRIPTION
**Pull request by khanchi97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ Table of Contents
   * [gitea.com](https://www.gitea.com/) - Unlimited public and private Git repos
   * [github.com](https://github.com/) — Unlimited public repositories and unlimited private repositories (with unlimited collaborators) with Pipelines for CI/CD
   * [gitlab.com](https://about.gitlab.com/) — Unlimited public and private Git repos with unlimited collaborators, with Pipelines for CI/CD, and 10 GB space per repo
+  * [heptapod.net](https://heptapod.net/) — Heptapod is a friendly fork of GitLab Community Edition providing support for Mercurial
```
Comment (ripienaar) [inline README.md:152]: Maybe better if the link went to https://foss.heptapod.net/

Comment (khanchi97) [reply README.md:152]: Yeah, you are probably right. I will change it.

Comment (khanchi97) [reply README.md]: Done.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**